public/js/components/Charts/AreaChartWrap.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
-import { Actions } from 'jumpstate';
 import { AreaChart , Themes } from 'formidable-charts';
 import { downloadCSV } from 'helpers/chartsHelpers'
 import ChartTheme from '../ChartTheme/theme';
 
 const customisedTheme = Object.assign({}, Themes.simple, ChartTheme);
 
-const AreaChartWrap = ({ data, xLabel, yLabel, yLabelStacked, isUpdating, scale, isStacked, error, hasToggle, height, titleHeader, chartType }) => {
+const AreaChartWrap = ({ data, xLabel, yLabel, yLabelStacked, isUpdating, scale, isStacked, error, hasToggle, height, titleHeader, chartType, actions }) => {
```

Comment:
Instead of passing down the actions, pass down an update function which is called on the level above